src/main/java/stream/flarebot/flarebot/FlareBot.java

```diff
@@ -383,6 +387,8 @@ protected void run() {
                 "Executed creations!", "Failed to execute creations!",
                 () -> manager.executeCreations());
 
+        futureActions = new ConcurrentHashSet<>();
```

Comment:
Not here 🤦‍♂️ 
I said final set for a reason. Put it where the declaration is.

Reply:
I'm a derp